tests/impl.cpp

```diff
@@ -5096,7 +5112,6 @@ result_t test_mm_max_pd(const SSE2NEONTestImpl &impl, uint32_t iter)
     __m128d a = load_m128d(_a);
     __m128d b = load_m128d(_b);
     __m128d c = _mm_max_pd(a, b);
-
```

Comment:
Don't do this. Minimize the necessary changes.

Reply:
Fixed.